actions/actions.py

```diff
@@ -643,17 +644,7 @@ def run(self, dispatcher, tracker, domain) -> List[EventType]:
             return []
 
         else:
-            # trigger get_started_step1 intent
-            return [
-                ActionExecuted("action_listen"),
-                UserUttered(
-                    "/get_started_step1",
-                    {
-                        "intent": {"name": "get_started_step1", "confidence": 1.0},
-                        "entities": [],
-                    },
-                ),
-            ]
```

Comment:
I'm confused about this, does `action_greet_user` usually happen after `get_started_step1`? shouldn't it go to the beginning of the onboarding flow?

Reply:
Yeah, I was confused too, but it looks like this is a kinda rare scenario where the user says "next step", before any steps have been completed, or maybe after all steps have been completed.

Reply:
e.g. start the conversation with "hi", then immediately after write "next step please"

Reply:
There's a lot of these kinds of stories: 
```
## Story from conversation with cfa8bb9deaf0427498c662745431a282 on December 15th 2018
* get_started_step1
    - action_greet_user
    - slot{"shown_privacy":true}
* chitchat
    - respond_chitchat
* enter_data
    - utter_not_sure
    - utter_possibilities
```